From 1cc78ecf2beb82dc1b6b796951cda7c244e761d3 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 20 Jun 2024 14:58:08 -0700
Subject: [PATCH] ==== maintainer profile update ====

Subject: Docs/damon/maintaier-profile: document a mailing tool and
community meetup series

There is a mailing tool that developed and maintained by DAMON
maintainer aiming to support DAMON community.  Also there are DAMON
community meetup series.  Both are known to have rooms of improvements
in terms of their visibility.  Document those on the maintainer's
profile document.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/O6ffnXpp | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/O6ffnXpp

diff --git a/damon_meta_changes/O6ffnXpp b/damon_meta_changes/O6ffnXpp
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

